index.html

```diff
@@ -15,7 +15,12 @@
      <header>
         <div class="container">
                 <a class="logo" href="#">My Folio</a>
-            <nav>
+                <div class="toggle_btn" id="jeg_mobile_toggle">
+                    <span></span>
+                    <span></span>
+                    <span></span>
```

Comment:
`span` elements indicate semantically that there is text there. If there's no text, we shouldn't be adding these here.